projects/robots/husarion/rosbot_xl/protos/RosbotXl.proto

```diff
@@ -0,0 +1,253 @@
+              0.000000e+00 0.000000e+00 0.000000e+00
+            ]
+          }
+        }
+      }
+      HingeJoint {
+        jointParameters HingeJointParameters {
+          axis 0.000000 1.000000 0.000000
+          anchor 0.085000 -0.135000 0.048000
+          dampingConstant 5.0
+          staticFriction 1.0
+        }
+        device [
+          RotationalMotor {
+            name "fr_wheel_joint"
+            maxVelocity 26.0
+            maxTorque 100.0
+          }
+          PositionSensor {
+            name "front right wheel motor sensor"
+          }
+        ]
+        endPoint Solid {
+          translation 0.085000 -0.135000 0.048000
+          rotation 0.000000 0.000000 1.000000 0.000000
+          children [
+            Transform {
+              translation 0.000000 0.000000 0.000000
+              rotation 0.000000 0.000000 1.000000 3.141592
+              children [
+                DEF WHEEL_SHAPE_A CadShape {
+                  url "meshes/mecanum_a.dae"
+                }
+              ]
+            }
+          ]
+          name "front right wheel"
+          boundingObject DEF RIGHT_WHEEL_CYLINDER Transform {
+            translation 0.000000 0.000000 0.000000
+            rotation 1.000000 0.000000 0.000000 -1.570800
+            children [
+              USE WHEEL_CYLINDER
+            ]
+          }
+          physics USE WHEEL_PHYSICS
+        }
+      }
+
+      HingeJoint {
+        jointParameters HingeJointParameters {
+          axis 0.000000 1.000000 0.000000
+          anchor -0.085000 0.135000 0.048000
+          dampingConstant 5.0
+          staticFriction 1.0
+        }
+        device [
+          RotationalMotor {
+            name "rl_wheel_joint"
+            maxVelocity 26.0
+            maxTorque 100.0
+          }
+          PositionSensor {
+            name "rear left wheel motor sensor"
+          }
+        ]
+        endPoint Solid {
+          translation -0.085000 0.135000 0.048000
+          rotation 0.000000 0.000000 1.000000 0.000000
+          children [
+            Transform {
+              translation 0.000000 0.000000 0.000000
+              rotation 0.000000 0.000000 1.000000 0.000000
+              children [
+                USE WHEEL_SHAPE_A
+              ]
+            }
+          ]
+          name "rear left wheel"
+          boundingObject DEF LEFT_WHEEL_CYLINDER Transform {
+            translation 0.000000 0.000000 0.000000
+            rotation 1.000000 0.000000 0.000000 -1.570800
+            children [
+              USE WHEEL_CYLINDER
+            ]
+          }
+          physics USE WHEEL_PHYSICS
+        }
+      }
+
+      HingeJoint {
+        jointParameters HingeJointParameters {
+          axis 0.000000 1.000000 0.000000
+          anchor -0.085000 -0.135000 0.048000
+          dampingConstant 5.0
+          staticFriction 1.0
+        }
+        device [
+          RotationalMotor {
+            name "rr_wheel_joint"
```

Comment:
same comments in this PROTO about device names

Reply:
Also here the name is the same as in [rosbot_xl_description](https://github.com/husarion/rosbot_xl_ros/blob/master/rosbot_xl_description/urdf/rosbot_xl_macro.urdf.xacro#L74)